migrations/20180212204016_create_export_job_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class CreateExportJobTable extends AbstractMigration
+{
+
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $this->table('export_job')
+            ->addColumn('user_id', 'integer', ['null' => false])
+            ->addColumn('entity_type', 'string', ['null' => false])
+            ->addColumn('fields', 'string', ['null' => true])
+            ->addColumn('filters', 'string', ['null' => true])
```

Comment:
Would filters include the survey(s) being requested? I believe that having a different job request for each survey might make sense, since the field list would potentially be vastly different for each CSV.

Reply:
about "different job for each survey" @kinstelli @willdoran => that's not part of the scope and needs to be user tested before/if we want to do it. Just an FYI because it's going to be more work for something we don't know for sure we want or how we want it

Reply:
@rowasc I think separating by surveys is in, no? Because otherwise the surveys will still look terrible, It's relatively straightforward to do. The filters would contain the surveys, I can just pull them out and split them up, though obviously it would spawn a large number of files if there was a large number of surveys....hmmmm....actually may be for this moment we should just output a single file then split it BUT we will need to split them up.

I'll see if there is an easy way to do this tomorrow.

Reply:
No, it wasn't in/scoped for this cycle because we need user testing. I also can't find an issue saying it should be chopped by survey. @willdoran 
It was considered something to do in a next step, because we want to know for sure how to do it/what the user expects.